src/Scene.js

```diff
@@ -34,10 +92,18 @@ class Scene extends React.Component<SceneProps, {}> {
                     <svg
                         xmlns='http://www.w3.org/2000/svg'
                         viewBox={`${minX} ${minY} ${width} ${height}`}>
-                        <RobotCharacter
-                            robotCharacterTransform={robotCharacterTransform}
-                            width={this.props.gridCellWidth * 0.8}
-                        />
+                        <defs>
+                            <clipPath id='Scene-clippath'>
+                                <rect x={minX} y={minY} width={width} height={height} />
+                            </clipPath>
+                        </defs>
+                        {this.drawGrid(width, height, minX, minY)}
```

Comment:
Let's change the parameter order to match our use of `rect` above:

    {this.drawGrid(minX, minY, width, height)}

Reply:
And to match the order of the values in the `viewBox`